wallet/crate/src/lib.rs

```diff
@@ -36,9 +36,10 @@ use manta_accounting::{
     wallet::{
         self,
         ledger::{self, ReadResponse},
-        signer::SyncData,
+        signer::{NetworkType as ActualNetworkType, SyncData},
```

Comment:
can you explain what the deal with this is?

Reply:
Just an alias for the networkType from manta-rs, I renamed it to cause less confusion